src-db/database/model/triggers/C_INVOICELINETAX_TRG.xml

```diff
@@ -163,12 +163,17 @@ BEGIN
       AND C_Tax_ID = :old.C_Tax_ID
       AND Recalculate = 'Y';
     END IF;
-
-    DELETE FROM C_INVOICETAX 
-    WHERE C_INVOICE_ID = :OLD.C_Invoice_ID
-    AND C_TAX_ID = :OLD.C_TAX_ID
-    AND ROUND(TAXBASEAMT, v_Prec) = 0
-    AND TAXAMT = 0;
+	
```

Comment:
Before the XML had spaces and now it has tabs, check it and try to have it unified (Only tabs or only spaces). The lines before and after the changes have only spaces